src/apps/safe-swap/SafeSwapApp.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import { BrowserRouter, HashRouter, Redirect, Route, Switch, useLocation, useRouteMatch } from 'react-router-dom'
+import { hot } from 'react-hot-loader/root'
+
+import { withGlobalContext } from 'hooks/useGlobalState'
+import useNetworkCheck from 'hooks/useNetworkCheck'
+import Console from 'Console'
+import { rootReducer, INITIAL_STATE } from './state'
+
+import { GenericLayout } from 'components/layout'
+import { Header } from './layout/Header'
+
+import { NetworkUpdater } from 'state/network'
+import { initAnalytics } from 'api/analytics'
+import RouteAnalytics from 'components/analytics/RouteAnalytics'
+import NetworkAnalytics from 'components/analytics/NetworkAnalytics'
+import { DIMENSION_NAMES } from './const'
+
+// Init analytics
+const GOOGLE_ANALYTICS_ID: string | undefined = process.env.GOOGLE_ANALYTICS_ID
+initAnalytics({
+  trackingCode: GOOGLE_ANALYTICS_ID,
+  dimensionNames: DIMENSION_NAMES,
+})
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Router: typeof BrowserRouter & typeof HashRouter = (window as any).IS_IPFS ? HashRouter : BrowserRouter
+
+const NotFound = React.lazy(
+  () =>
+    import(
+      /* webpackChunkName: "Extra_routes_chunk"*/
+      './pages/NotFound'
+    ),
+)
+
+const Home = React.lazy(
+  () =>
+    import(
+      /* webpackChunkName: "Trade_chunk"*/
+      './pages/Swap'
+    ),
+)
+
+/**
+ * Update the global state
+ */
+export function StateUpdaters(): JSX.Element {
```

Comment:
Minor nitpick: I don't think this needs to be exported

Reply:
u right. I also removed the export in the explorer